src/components/Common/UI/Header/Header.styles.ts

```diff
@@ -3,16 +3,15 @@ import { colors } from "~/constants";
 import { flex } from "~/styles/utils/flex";
 import { pxToRem } from "~/utils";
 
-export const Header = styled.header`
+export const Header = styled.header<{ bgColor?: string }>`
   position: fixed;
   z-index: 10;
 
   ${flex({ justifyContent: "space-between", alignItems: "center" })}
   width: 100%;
   height: ${pxToRem(50)};
   padding: 0 ${pxToRem(18)};
-
-  background: ${colors.grayf8};
+  background: ${({ bgColor }) => bgColor || colors.grayf8};
```

Comment:
```suggestion
  background: ${({ bgColor =  colors.grayf8 }) => bgColor};
```

요건 어떠신가요? 불필요한 제어문이 없어집니다

Reply:
엇 이렇게 바꾸겠습니다